x/zoneconcierge/keeper/ibc_channels.go

```diff
@@ -0,0 +1,55 @@
+package keeper
+
+import (
+	"github.com/babylonchain/babylon/x/zoneconcierge/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	channeltypes "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
+)
+
+func (k Keeper) GetAllChannels(ctx sdk.Context) []channeltypes.IdentifiedChannel {
+	return k.channelKeeper.GetAllChannels(ctx)
+}
+
+// GetAllOpenZCChannels returns all open channels that are connected to ZoneConcierge's port
+func (k Keeper) GetAllOpenZCChannels(ctx sdk.Context) []channeltypes.IdentifiedChannel {
```

Comment:
```suggestion
func (k Keeper) GetAllOpenChannels(ctx sdk.Context) []channeltypes.IdentifiedChannel {
```
Given that we're already on the zoneconcierge module. Also matches with the other names here that do not have the "ZC" prefix.

Reply:
I'm not sure whether Babylon will use IBC channels for other use cases. If yes, then there can be IBC channels that are not binded to ZoneConcierge's port and support use cases other than ZoneConcierge's. This function aims at filtering channels that bind ZoneConcierge's port (thus support ZoneConcierge's use cases). So may the `ZC` here is necessary?

Reply:
Actually not. Here `GetAllOpenZCChannels` gets channels by using `channelKeeper.GetAllChannels(ctx)`, which returns all channels, no matter which port they are binding or which module they are associated with. That's why we filter channels not only by their status (i.e., open channels) but also port they are binding.